KubeArmor/core/kubeUpdate.go

```diff
@@ -190,6 +190,9 @@ func (dm *KubeArmorDaemon) UpdateEndPointWithPod(action string, pod tp.K8sPod) {
 
 		newPoint.NamespaceName = pod.Metadata["namespaceName"]
 		newPoint.EndPointName = pod.Metadata["podName"]
+		newPoint.Owner.Ref = pod.Metadata["owner.controller"]
+		newPoint.Owner.Name = pod.Metadata["owner.controllerName"]
+		newPoint.Owner.Namespace = pod.Metadata["namespaceName"]
```

Comment:
This namespace needs to be picked from `owner`.

Reply:
updated